src/write_resume_data.cpp

```diff
@@ -232,7 +232,7 @@ namespace libtorrent {
 			// write file priorities
 			entry::list_type& prio = ret["file_priority"].list();
```

Comment:
the more significant improvement here would probably be a ``prio.reserve()`` call